for_loop.py

```diff
@@ -37,43 +64,74 @@ def reverse_list(l):
     """
     Returns a new list reversing the elements in the list l.
     """
-    return l
+    index = 0
```

Comment:
Is `sort_list()` working? This code seems wrong to me. If you haven't done the function yet, please don't commit it.